docs/release_notes.rst

```diff
@@ -173,8 +173,8 @@ Data Coverage
 
   * :ref:`core_ferc714__respondent_id` (linking FERC-714 respondents to EIA utilities)
   * :ref:`core_ferc714__hourly_demand_pa` (hourly electricity demand by planning area)
-  * :ref:`fipsified_respondents_ferc714` (annual respondents with county FIPS IDs)
-  * :ref:`summarized_demand_ferc714` (annual demand for FERC-714 respondents)
+  * :ref:`out_ferc714__fipsified_respondents` (annual respondents with county FIPS IDs)
```

Comment:
non-blocking: `respondents_with_fips` might be clearer

Reply:
I agree, but I want to preserve the `PudlTabl` API for now.

Reply:
It should be possible to change the underlying table asset name and keep the PudlTabl method name as it was without altering access (the same as for all the other assets). In which case we'd still want to update the release notes to refer to the actual table.